docs/tutorial/tricks.md

```diff
@@ -0,0 +1,141 @@
+# Common tricks used in (T)FHE
+
+As explained in the [Basics of FHE](../getting-started/fhe_basics.md), the challenge for a developer
+is to adapt his code to fit FHE constraints. In this document we have collected some common examples
+to illustrate the kind of optimization one can do to get better performances.
```

Comment:
I think it's `performance` not `performances`.

Reply:
fixed